src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -1755,44 +1773,53 @@ def hadamard_matrix(n, existence=False, check=True):
                        for i in range(R)])
         M = chad1.stack(chad2)
     elif is_prime_power(n - 1) and (n - 1) % 4 == 3:
+        name = "paleyI" + name
```

Comment:
As above, here I think it should be `"paleyI "`.

Reply:
done